pkgs/development/python-modules/logica/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, buildPythonPackage, fetchPypi, isPy27 }:
+
+buildPythonPackage rec {
+  pname = "logica";
+  version = "1.3.13";
+
+  disabled = isPy27;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "72c764fa5c20ee280af7386e739be2430c67832da59f12aa458bdca491866de1";
+  };
+
+  # PyPI tarball doesn't ship README.md
+  postPatch = ''
+    touch logica/README.md
+  '';
+
```

Comment:
```suggestion

  checkPhase = ''
    ${python.interpreter} run_all_tests.py
  '';
```

Reply:
Thanks for reviewing, I added this checkPhase. 
Unfortunately, the pypi tarball is missing some tests files, I'm going to report it to upstream later this week.